docs/apis/avalanchego/apis/x-chain.md

```diff
@@ -791,9 +791,9 @@ curl -X POST --data '{
     "id"     :1,
     "method" :"avm.getAssetDescription",
     "params" :{
-        "assetID" :"2fombhL7aGPwj3KH4bfrmJwW6PVnMobf9Y2fn9GwxiAAJyFDbe"
+        "assetID" :"FvwEAhmxKfeiG8SnEvq42hc6whRyY3EFYAvebMqDNDGCgxN5Z"
     }
-}' -H 'content-type:application/json;' 127.0.0.1:9650/ext/bc/X
+}' -H 'content-type:application/json;' https://api.avax.network/ext/bc/X
```

Comment:
all other end points are `127.0.0.1:9650/ext/bc/X` in this doc. 127.0.0.1 will still work if the local node is on mainnet.